calico-policies/private-network-isolation/calico-v3/us-south/allow-private-services-pods.yaml

```diff
@@ -74,6 +74,29 @@ spec:
       - 514
     protocol: UDP
     source: {}
+  - action: Allow
+    destination:
+      nets:
+      # IBM ATracker API : Prod US South
+      - 166.9.58.136/32
+      - 166.9.48.211/32
+      - 166.9.51.140/32
+      # IBM ATracker API : Stage US South
+      - 166.9.51.58/32
+      - 166.9.48.161/32
+      - 166.9.58.81/32
+      # IBM ATracker Ingester : Prod US South
```

Comment:
Why does this PR only include a few regions?

Reply:
Currently ATracker is only operating in these regions. We will add IPs for new regions if required later.